### ISSUE TITLE
USER STORY: Predict Task Success notebook - Refit ML pipeline using only the two most important variables

### ISSUE THREAD

**Issue by Markus-Hefner** [Markus-Hefner/Portfolio-Project-5#13]
As a **client** I can **read through the notebook** so that **I can learn how the model performed with only the two most important variables were**. 

**Acceptance criteria** 

AC1 - A model to predict the task success of a developer given all variables
AC2 - The pipeline responsible for Data Cleaning and Feature Engineering is saved as a file and pushed to the repo
AC3 - The pipeline responsible for Feature Scaling, and Model is saved as a file and pushed to the repo
AC4 - The Feature Importance Plot is saved as a file and pushed to the repo

**Tasks**

- [x] Rewrite ML pipeline for Data Cleaning and Feature Engineering
- [x] Rewrite ML Pipeline for Modelling
- [x] Split Train Test Set with only the best features
- [x] Check results with most suitable algorithm and best hyperparameter configuration
- [x] Assess feature importance
- [x] Evaluate pipeline on train and test set
- [x] Create files as mentioned in AC2, AC3 and AC4
- [x] Push everything to repo
